docs/useShowController.md

```diff
@@ -51,13 +51,56 @@ const PostShow = () => {
 
 This custom Show view has no action buttons - it's up to you to add them in pure React.
 
+**Tip**: Use [`<ShowBase>`](./ShowBase.md) instead of `useShowController` if you need a component version of that hook.
+
 ## Parameters
```

Comment:
```suggestion
## Props
```

Reply:
A hook doesn't have props.

Reply:
Right! My bad

Reply:
The header in the parameters table still says "Prop", just saying :roll_eyes: :grin: